components/about/missionSection.js

```diff
@@ -35,7 +35,7 @@ const MissionSection = () => (
               <img
                 alttext="Fall 2018 team photo"
                 style={props}
-                src="/static/images/sp19-team.jpg"
+                src="/static/images/fa19-team.jpg"
```

Comment:
This pic is a little weird b/c the left half is goofy and the right half isn't. I'd try the regular picture or the one where whoever took the picture was standing up so that there's no grass. Try both and put screenshots of both in ur PR so we can pick the one that works. I think if everyone was goofy in the pic it might've worked.